Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.16-alpine as build
+
+ARG LOFTSMAN_VERSION=dev
+ARG HELM_VERSION=3.5.4
+
+ENV GO111MODULE=on
+
+RUN apk add --no-cache git make bash
+RUN git clone https://github.com/helm/helm.git /helm
+WORKDIR /helm
+RUN git checkout v${HELM_VERSION}
+RUN make
```

Comment:
any reason we make the helm binary ourselves instead of downloading an official release? Maybe add some comments around why we took this approach.

Reply:
yeah, I'm not sure I trust go binaries built outside of Alpine to run consistently well in Alpine, so actually doing the go building within the container that'll run it. Suggest away on other things, alternative base image, etc. Good call on the note regardless, I'll do that for now at least.